### DISCUSSION TITLE
Option to run (F5) app on all selected platforms in one go simultaneously - .NET Maui

### DISCUSSION THREAD

**Discussion by MGohil** [dotnet/maui#14968] [Ideas]
Currently we are able to run the app in only one platform simulator at a time. This means, for each small change, we have to check in different platform one after another, eating time.
This was possible in Xamarin.Forms where we had different projects per platform and we could select multiple projects to run at a time. But, in .Net Maui, having a single project for all platforms, there seems no such feature available.

To improve the productivity, it is good to have a feature where we select target platform simulators/devices and if we run the project, it deploys on all those target platform simulators at the same time and we can test our development things on the one go on each multiple platforms we support.
Also, the Hot Reload should work in all simulators running the current run of the app.

This will help gain huge productivity.

Thanks.

**Comment by Pastajello** [comment]
Do we have any news on this? This is very helpful and actually quite easy to setup in XF.
